src/Http/Headers/src/HeaderNames.cs

```diff
@@ -6,6 +6,8 @@ namespace Microsoft.Net.Http.Headers
     /// <summary>
     /// Defines constants for well-known HTTP headers.
     /// </summary>
+    // Modification policy: This list is not intended to be exhaustive, it primarily contains values used by the framework itself.
```

Comment:
```suggestion
    // MODIFICATION POLICY: This list is not intended to be exhaustive, it primarily contains values used by the framework itself.
```

Really emphasize the point 😄